package.json

```diff
@@ -72,8 +72,8 @@
     "js-yaml": "^3.13.1"
   },
   "volta": {
-    "node": "12.4.0",
-    "yarn": "1.22.4"
+    "node": "14.18.3",
```

Comment:
I use volta and upgraded this, not sure if it impacts anyone or CI or something else?